### ISSUE TITLE
sentry-cli command not found

### ISSUE THREAD

**Issue by ChristianVermeulen** [mathieu-bour/setup-sentry-cli#4]
For some reason, my runner still can not find my sentry cli command...

My github action config:
```
	- name: Setup Sentry CLI
        uses: mathrix-education/setup-sentry-cli@0.1.0
        with:
          version: latest # optional if 'latest'
          url: https://sentry.io
          token: ${{ secrets.SENTRY_AUTH_TOKEN }} # from GitHub secrets
          organization: itexperts
          project: webreporttool
```

The result in the action runner (which looks good to me):
![Screenshot 2020-09-30 at 10 45 41](https://user-images.githubusercontent.com/1062751/94663816-6987dd00-030a-11eb-8ebb-ad7b59d89626.png)

When actually calling the `sentry-cli` command:
![Screenshot 2020-09-30 at 10 46 01](https://user-images.githubusercontent.com/1062751/94663875-7a385300-030a-11eb-8272-a2c919a76b35.png)

Am I missing something here?

**Comment by matbour** [comment]
@ChristianVermeulen Please update to the latest version 1.0.0, it may be related to [GHSA-mfwh-5m23-j46w](https://github.com/actions/toolkit/security/advisories/GHSA-mfwh-5m23-j46w). I'm closing this one for now, feel free to re-open if necessary!

**Comment by AtiqGauri** [comment]
@mathieu-bour 

got this exact same issue while using action on mac-runner... works perfectly fine on windows-runner
I am on 1.0.0 version

**Comment by matbour** [comment]
I'll take a look on this one, seems that this repo need a bit of polish.

**Comment by matbour** [comment]
This problem should be fixed in mathieu-bour/setup-sentry-cli@1.1.0.
Feel free to open a new issue @AtiqGauri if you have any further problem.
Thank you for using my action!

**Comment by nmoinvaz** [comment]
I experienced a similar command not found issue running the `setup-sentry-cli` action inside a separate "setup" composite action and trying to access `sentry-cli` from within my main workflow. I worked around it by moving `setup-sentry-cli` action into my main workflow.
